### PR TITLE
AEA-2326-Test-data-changes

### PR DESCRIPTION
**Pull request by TomSmithCoding**

### CODE REVIEW

File: coordinator/tests/resources/test-data/practitioner-role/index.ts [hunk 1]
```diff
@@ -1,7 +1,6 @@
 import {fhir} from "@models"
 import {telecom} from "../telecom"
-export * from "./dispense-notification"
-export * from "./task"
+export * from "./pr-org-ref"
```
Comment (JackRobinsonIW) [inline coordinator/tests/resources/test-data/practitioner-role/index.ts:3]: I think we might want a different name here, this filename is confusing

File: coordinator/tests/debug/conversions.spec.ts [hunk 2]
```diff
@@ -11,7 +11,7 @@ afterAll(() => {
 })
 
 const allExpectedPassingConvertExamples = fetcher.convertExamples.filter(e => e.isSuccess)
-test.skip.each(allExpectedPassingConvertExamples)(
+test.each(allExpectedPassingConvertExamples)(
```
Comment (dan-jones-iw) [inline coordinator/tests/debug/conversions.spec.ts:14]: undo!!!!!!!!!!!!!!!!!!!!!

Comment (TomSmithCoding) [reply coordinator/tests/debug/conversions.spec.ts:14]: !!!!!!! Noooooooooo! I can't believe I've done this

**Review by dan-jones-iw — APPROVED** [review]

**Pull request merged**